components/user/UserLink.js

```diff
@@ -24,15 +39,22 @@ export default function UserLink({
       href={`${BASE_URL}/api/profiles/${username}/links/${link._id}`}
       target="_blank"
       rel="noopener noreferrer"
-      className={`rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40 grow ${
+      className={`relative rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium ${ link.animation === "Glow Icon" ? "z-0" : "" } ${ link.animation !== "Glow" ? "dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40" : ""} grow ${
```

Comment:
If it's more convenient in this case, it may be an option using classNames util

Reply:
I don't know about classNames util can you give an example please?